doc/connectivity/bluetooth/bluetooth-ctlr-arch.rst

```diff
@@ -0,0 +1,261 @@
+.. _bluetooth-ctlr-arch:
+
+Bluetooth Low Energy Controller
+###############################
+
+Overview
+********
```

Comment:
This section is empty. Either add something or remove it please

Reply:
Added an overview diagram.